<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
  <class>NeuroEditor</class>
  <widget class="QMainWindow" name="NeuroEditor">
    <property name="geometry">
      <rect>
        <x>0</x>
        <y>0</y>
        <width>1000</width>
        <height>720</height>
      </rect>
    </property>
    <property name="windowTitle">
      <string>NeuroEditor</string>
    </property>

    <widget class="QWidget" name="centralWidget"/>
    <widget class="QMenuBar" name="menuBar">
      <property name="geometry">
        <rect>
          <x>0</x>
          <y>0</y>
          <width>1000</width>
          <height>23</height>
        </rect>
      </property>
      <widget class="QMenu" name="menuFile">
        <property name="title">
          <string>File</string>
        </property>
        <addaction name="actionOpenSWCFile"/>
        <addaction name="actionSaveSWCFile"/>
        <addaction name="actionExportMesh"/>
        <addaction name="actionQuit"/>
      </widget>
      <widget class="QMenu" name="menuOptions">
        <property name="title">
          <string>Options</string>
        </property>
        <addaction name="actionView"/>
        <addaction name="actionSelect"/>
        <addaction name="actionEdit"/>
        <addaction name="actionCorrect"/>
      </widget>
      <widget class="QMenu" name="menuHelp">
        <property name="title">
          <string>Help</string>
        </property>
        <addaction name="actionAbout"/>
      </widget>
      <addaction name="menuFile"/>
      <addaction name="menuOptions"/>
      <addaction name="menuHelp"/>
    </widget>



    <widget class="QStatusBar" name="statusbar"/>
    <widget class="QToolBar" name="toolBar">
      <property name="windowTitle">
        <string>toolBar</string>
      </property>
      <attribute name="toolBarArea">
        <enum>TopToolBarArea</enum>
      </attribute>
      <attribute name="toolBarBreak">
        <bool>false</bool>
      </attribute>
      <addaction name="actionOpenSWCFile"/>
      <addaction name="actionSaveSWCFile"/>
      <addaction name="actionExportMesh"/>
      <addaction name="actionUndo"/>
      <addaction name="actionView"/>
      <addaction name="actionSelect"/>
      <addaction name="actionEdit"/>
      <addaction name="actionCorrect"/>
    </widget>



    <action name="actionQuit">
      <property name="icon">
        <iconset resource="../qrc/resources.qrc">
        <normaloff>:/icons/system-log-out.png</normaloff>:/icons/system-log-out.png</iconset>
      </property>
      <property name="text">
        <string>Quit</string>
      </property>
    </action>
    <action name="actionOpenSWCFile">
      <property name="icon">
        <iconset resource="../qrc/resources.qrc">
        <normaloff>:/icons/document-open.png</normaloff>:/icons/document-open.png</iconset>
      </property>
      <property name="text">
        <string>Open SWC</string>
      </property>
      <property name="toolTip">
        <string>Open SWC</string>
      </property>
    </action>
    <action name="actionSaveSWCFile">
      <property name="icon">
        <iconset resource="../qrc/resources.qrc">
        <normaloff>:/icons/document-save.png</normaloff>:/icons/document-save.png</iconset>
      </property>
      <property name="text">
        <string>Save SWC</string>
      </property>
      <property name="toolTip">
        <string>Save SWC</string>
      </property>
    </action>
    <action name="actionExportMesh">
      <property name="icon">
        <iconset resource="../qrc/resources.qrc">
        <normaloff>:/icons/document-save-as.png</normaloff>:/icons/document-save-as.png</iconset>
      </property>
      <property name="text">
        <string>Export mesh</string>
      </property>
      <property name="toolTip">
        <string>Export mesh</string>
      </property>
    </action>
    <action name="actionView">
      <property name="checkable">
        <bool>true</bool>
      </property>
      <property name="text">
        <string>View</string>
      </property>
      <property name="toolTip">
        <string>View</string>
      </property>
    </action>
    <action name="actionSelect">
      <property name="checkable">
        <bool>true</bool>
      </property>
      <property name="text">
        <string>Select</string>
      </property>
      <property name="toolTip">
        <string>Select</string>
      </property>
    </action>
    <action name="actionEdit">
      <property name="checkable">
        <bool>true</bool>
      </property>
      <property name="text">
        <string>Edit</string>
      </property>
      <property name="toolTip">
        <string>Edit</string>
      </property>
    </action>
    <action name="actionCorrect">
      <property name="checkable">
        <bool>true</bool>
      </property>
      <property name="text">
        <string>Correct</string>
      </property>
      <property name="toolTip">
        <string>Correct</string>
      </property>
    </action>
    <action name="actionAbout">
      <property name="text">
        <string>About ...</string>
      </property>
      <property name="toolTip">
        <string>About ...</string>
      </property>
    </action>
    <action name="actionUndo">
      <property name="icon">
        <iconset resource="../qrc/resources.qrc">
        <normaloff>:/icons/edit-undo.png</normaloff>:/icons/edit-undo.png</iconset>
      </property>
      <property name="text">
        <string>Undo</string>
      </property>
      <property name="toolTip">
        <string>Undo trace changes</string>
      </property>
    </action>

  </widget>
  <resources/>
  <connections/>
</ui>
